README.md

```diff
@@ -89,6 +89,7 @@ A full list supported options are listed here.
 |       video-paths      |           -V           | A list of videos that will be saved in the simulators.                              |     N    | n/a              |
 |       image-paths      |           -I           | A list of images that will be saved in the simulators.                              |     N    | n/a              |
 | unsafe-skip-xcode-version-check |               | Skip Xcode version check                                                            |     N    | NO               |
+|  retry-app-crash-tests |                        | Retry tests that crashed app and consider it non-fatal if it passes on retry.       |     N    | false            |
```

Comment:
Do we really need another flag? Isn't this what `error-retries` should be used for?

Reply:
This flag -- `retry-app-crash-tests` -- is to enable/disable retries on app crashes.
The `error-retries` flag on the other hand is to configure the number of total retries allowed, which is close to `failure-tolerance` flag. We can think of merging `failure-tolerance` and `error-retries`.